components/infobox/wikis/mobilelegends/infobox_item_custom.lua

```diff
@@ -294,4 +286,18 @@ function CustomItem._getAttributeCells(attributeCells)
 	return widgets
 end
 
+function CustomItem._manaLossDisplay()
+	local manaLoss = _args['mana loss'] or _args.manaloss
```

Comment:
New wiki, shouldn't it be fine on deciding on one of them instead of having both?

Reply:
true, and that allows us to kick the entire function